modules/gapi/samples/pipeline_builder_tool/test_pipeline_builder.py

```diff
@@ -0,0 +1,893 @@
+import os
```

Comment:
@alalek Is is possible to run these tests as part of precommit job?

Reply:
Only functional tests with filtering capability are launched.
We don't test tools or samples. No plans to change that.

Reply:
Got it, thanks!